explorer/explorermiddleware.go

```diff
@@ -200,3 +206,34 @@ func ProposalPathCtx(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r.WithContext(ctx))
 	})
 }
+
+// MenuFormParser parses a form submission from the navigation menu.
+func MenuFormParser(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.FormValue(darkModeFormKey) != "" {
+			cookie, err := r.Cookie(darkModeCoookie)
+			if err != nil && err != http.ErrNoCookie {
+				log.Errorf("Cookie dcrdataDarkBG retrieval error: %v", err)
+			} else {
+				if err == http.ErrNoCookie {
+					cookie = &http.Cookie{
+						Name:   darkModeCoookie,
+						Value:  "1",
+						MaxAge: 0,
+					}
+				} else {
+					cookie.Value = "0"
```

Comment:
At first I was thinking this should be a toggle, but I see that no cookie is set in `commonData` if the dark mode cookie value is not "1".  Seems good, but I'm just curious if the idea was to limit cookie use to only when dark mode is enabled, because it seems like the cookie could store both on or off values.

Reply:
Either way is fine. My thoughts are that we will always have to check for the existence of the cookie first anyway, with a missing cookie yielding some default behavior. Since we have to check anyway, it might save a step to delete the cookie. Do you prefer the other way?

Reply:
Nope, I like your way.  Was just wondering if I was following the intended logic.